.taplo.toml

```diff
@@ -5,3 +5,9 @@ align_entries       = true
 allowed_blank_lines = 1
 reorder_arrays      = true
 reorder_keys        = true
+
+[[rule]]
+keys = ["build-dependencies", "dependencies", "dev-dependencies"]
```

Comment:
We usually have `dev-dependencies` before `dependencies`. Why have those not been changed here? I don't see any shuffling around. But I would say `dev-dependencies` should come first, since they don't have any feature nor optional ones?

Reply:
It is not reordering the keys, only applying the rules to the keys